clients/TypeScript/packages/schema/src/index.ts

```diff
@@ -8,53 +8,11 @@
 /**
  * A Cardano Block, which can take different forms depending on the era.
  */
-export type Block = Byron | Shelley | Allegra | Mary | Alonzo;
```

Comment:
I am not sure to understand why there are so many changes here :thinking: ? Wasn't the schema up-to-date already? Or perhaps did you use a more recent version of json-schema-to-typescript?

Any clue?